BinaryEncoding.md

```diff
@@ -786,9 +783,138 @@ for [future :unicorn:][future multiple tables] use and must be 0 in the MVP.
+| `i32.atomic_or16_u` | `0xe615` | `3` `memory immediate` | atomic or with memory |
+| `i64.atomic_or8_s` | `0xe616` | `3` `memory immediate` | atomic or with memory |
+| `i64.atomic_or8_u` | `0xe617` | `3` `memory immediate` | atomic or with memory |
+| `i64.atomic_or16_s` | `0xe618` | `3` `memory immediate` | atomic or with memory |
+| `i64.atomic_or16_u` | `0xe619` | `3` `memory immediate` | atomic or with memory |
+| `i64.atomic_or32_s` | `0xe61a` | `3` `memory immediate` | atomic or with memory |
+| `i64.atomic_or32_u` | `0xe61b` | `3` `memory immediate` | atomic or with memory |
+| `i32.atomic_xor` | `0xe610` | `4` `memory immediate` | atomic xor with memory |
+| `i64.atomic_xor` | `0xe611` | `4` `memory immediate` | atomic xor with memory |
+| `i32.atomic_xor8_s` | `0xe612` | `4` `memory immediate` | atomic xor with memory |
+| `i32.atomic_xor8_u` | `0xe613` | `4` `memory immediate` | atomic xor with memory |
+| `i32.atomic_xor16_s` | `0xe614` | `4` `memory immediate` | atomic xor with memory |
+| `i32.atomic_xor16_u` | `0xe615` | `4` `memory immediate` | atomic xor with memory |
+| `i64.atomic_xor8_s` | `0xe616` | `4` `memory immediate` | atomic xor with memory |
+| `i64.atomic_xor8_u` | `0xe617` | `4` `memory immediate` | atomic xor with memory |
+| `i64.atomic_xor16_s` | `0xe618` | `4` `memory immediate` | atomic xor with memory |
+| `i64.atomic_xor16_u` | `0xe619` | `4` `memory immediate` | atomic xor with memory |
+| `i64.atomic_xor32_s` | `0xe61a` | `4` `memory immediate` | atomic xor with memory |
+| `i64.atomic_xor32_u` | `0xe61b` | `4` `memory immediate` | atomic xor with memory |
+| `i32.atomic_xchg` | `0xe610` | `5` `memory immediate` | atomic exchange with memory |
+| `i64.atomic_xchg` | `0xe611` | `5` `memory immediate` | atomic exchange with memory |
+| `i32.atomic_xchg8_s` | `0xe612` | `5` `memory immediate` | atomic exchange with memory |
+| `i32.atomic_xchg8_u` | `0xe613` | `5` `memory immediate` | atomic exchange with memory |
+| `i32.atomic_xchg16_s` | `0xe614` | `5` `memory immediate` | atomic exchange with memory |
+| `i32.atomic_xchg16_u` | `0xe615` | `5` `memory immediate` | atomic exchange with memory |
+| `i64.atomic_xchg8_s` | `0xe616` | `5` `memory immediate` | atomic exchange with memory |
+| `i64.atomic_xchg8_u` | `0xe617` | `5` `memory immediate` | atomic exchange with memory |
+| `i64.atomic_xchg16_s` | `0xe618` | `5` `memory immediate` | atomic exchange with memory |
+| `i64.atomic_xchg16_u` | `0xe619` | `5` `memory immediate` | atomic exchange with memory |
+| `i64.atomic_xchg32_s` | `0xe61a` | `5` `memory immediate` | atomic exchange with memory |
+| `i64.atomic_xchg32_u` | `0xe61b` | `5` `memory immediate` | atomic exchange with memory |
+
+## Atomic compare exchange operators ([described here](Semantics.md#atomic-memory-accesses))
+
+| `i32.atomic_cmpxchg` | `0xe61c` | `memory immediate` | atomic compare exchange with memory |
```

Comment:
The formatting on this table is broken, possible because of the lack of a heading.